rfc-016-slack-channel-names.md

```diff
@@ -0,0 +1,21 @@
+# Slack channel names
+
+## Summary
+
+Guidelines for slack channel naming conventions building on [Slack's own guidelines](https://get.slack.help/hc/en-us/articles/217626408-Create-guidelines-for-channel-names).
+
+## Problem
+
+There is no standard naming convention for Slack channel names and as we grow they become harder to discover. It's difficult to understand whether a channel is for discussion on topics or with teams. For example some SRE questions go into #live-services, some into #help, #engineering and even into #announcements.
+
+## Proposal
+
+- You SHOULD create discussion channels for particular engineering topics such as Kubernetes, React, etc.
+- You SHOULD create team channels for all teams within the business whether delivery teams, sales, etc.
+- You SHOULD create project channels for particular team projects that have a focussed goal, an example would be Homes HIF is a project of the Homes team.
+- You MAY create collaboration channels and invite customers to them if you want to use Slack as a communication tool.
+
+- You MUST prefix engineering topics with `#eng-`. For example, `#eng-k8s`, `#eng-react`, `#eng-tdd`.
+- You MUST prefix team channels with `#team-`. For example `#team-dmc`, `#team-sales`, `#team-people`, `#team-homes`, `#team-sre`.
+- You MUST prefix project channel with `#proj-`. For example `#proj-dmc-watg`, `#proj-homes-hif`, `#proj-productionisation`.
```

Comment:
what is a project channel vs a team channel

Reply:
I tried to describe this above.  Projects are goal focussed and owned by teams. Teams may choose not to have specific project rooms but some already do. See `#he-strategy`, `#he-coaching`, `#he-hevm`.

Reply:
And the other secret small groups that are interest only.

Reply:
> Going to through the Live Services spanner in the works here - are we a project, engineering, team, or our (usual) own brand of misc?
> 
> We want LS to remain an open place where delivery teams can come and ask SRE related questions, so team doesn't fit. We're also going to be sprinting alongside Delivery teams int he future but that will change on a weekly basis, so project doesn't fit.

I noticed @lukemorton may have already rebranded us `#team-sre` 😉

Reply:
@booyaa I did ;) You might say p15n is a project: `#proj-sre-p15n`

Reply:
can we change this to current branding please (`team-live-services` vs `team-sre`)

Reply:
Unsure on if this would fall under `proj` or not.

Within LS, we generate channels whenever a p1 or code black occurs, and discussions on slack need to take place.

our current convention has been `#<customer>-incident-<project>-<incident nature>`, (eg `#akzo-incident-efh-k8s`).

Would `proj-akzo-incdent-efh-k8s` be used in this case, or can we introduce a new prefix? Perhaps `fire-<team>-[<proj>-]<nature>`?

Reply:
Feels like we are creating prefixes within prefixes. Not sure that's a solution there.

Reply:
Unsure on which structure you prefer @EThom73

Reply:
Thank you @lukemorton for posting my comments while github was broken for me

Reply:
Were we all approved with `#fire-` prefix?

Reply:
Fire is used to refer to a specific incident on the SRE incident escalation process (code red). There is a stage above that that can come into play - Code Black. 

It's a minor point but generally an incident channel will only be created in the case of  Code Black which is above fire.

Reply:
After speaking with Anthony, will keep this RFC ignorant of Live Service specific channels for now. Future RFC can specify this if necessary.